README.md

```diff
@@ -1,6 +1,6 @@
 # Snorkeling-Full-Text
 
-This is an upgraded version of the original [snorkeling project](https://github.com/greenelab/snorkeling). 
+This is an upgraded version of the original [snorkeling project](https://github.com/greenelab/snorkeling).
```

Comment:
Sorry its been a while, so here you're using full text vs previously using abstracts to learn relationships (like disease gene relationships). Where you I recall you have an input X matrix that is abstract x label function and the Y = 1/0 there is a relationship

Reply:
Also super nit-picky. Just noticed a few typos in your old repo readme if you feel like updating it

Reply:
Whats the reason this repo is being deprecated? Is full text just much better?

Reply:
>Sorry its been a while, so here you're using full text vs previously using abstracts to learn relationships (like disease gene relationships). Where you I recall you have an input X matrix that is abstract x label function and the Y = 1/0 there is a relationship

Yes for this section its individual sentences from abstracts X label function input. Once I have this matrix I'm feeding it into the generative model to estimate "accurate" training labels I can use for downstream machine learning (a few PRs from now).

>Also super nit-picky. Just noticed a few typos in your old repo readme if you feel like updating it

Feel free to suggest changes or pass them on to me. Appreciate the proof reading.

>Whats the reason this repo is being deprecated? Is full text just much better? 

That's what this new angle is going to find out. Given the programming bugs in my last repository, I decided that just starting fresh was a better option. I had to go all the way back to square one, so depreciating the old repo just made sense in my head.

Reply:
Just curious, I understand the logic for starting fresh, but is there a reason that you're not using abstracts here again..or was that part of the previous issue?

Reply:
>Just curious, I understand the logic for starting fresh, but is there a reason that you're not using abstracts here again..or was that part of the previous issue?

No I am using abstracts as well. Its just that pubtator updates every month or so. So my previous repo is outdated by a few years.